app/views/administrate/application/index.html.erb

```diff
@@ -57,10 +53,9 @@ It renders the `_table` partial to display details about the resources.
     "collection",
     collection_presenter: page,
     collection_field_name: resource_name,
-    page: page,
-    resources: resources,
+    collection_names: [resource_name],
```

Comment:
How about renaming `collection_names` as `allowed_order_params`. At the end of the day, that's exactly what it's used for: telling `#sanitised_order_params` which other params can be permitted.

The mere existence of this param is awkward, but it's not the fault of your PR. Search params are sanitized at the bottom (templates/helpers), with information about what's allowed flowing top-to-bottom
  1. Dashboard/template figures out which params it allows.
  2. It renders partial and tells it "please allow these params along with those you allow"

I wonder if this should be sanitized a the top (the controller), with this info flowing bottom-to-top:
  1. Dashboards need to publish which params they allow
  2. Controller asks for lists, then sanitises params before rendering the top template.
  3. Templates don't need to know about sanitising.

But this is an issue for another day, out of scope for this PR.